### PR TITLE
update copyright statements

### PR DESCRIPTION
**Pull request by rosemarymarano**
As per Monica, "“Every *.java,* .sh, Dockerfile, yaml, *.py file that gets added to our open source projects (operator, monitoring Exporter, logging Exporter, image tool,  WDT), no matter whether if it is for testing or main source code, needs to have the Copyright and UPL license statement in order for us to be compliant.”

### CODE REVIEW

File: core/src/test/resources/encryption-test-variables.yaml [hunk 1]
```diff
@@ -1,3 +1,6 @@
+# Copyright 2019, Oracle Corporation and/or its affiliates. All rights reserved.
```
Comment (ddsharpe) [inline core/src/test/resources/encryption-test-variables.yaml]: These domain YAML files do NOT support comments or blank lines.

File: core/src/test/resources/encryption-test.yaml [hunk 2]
```diff
@@ -1,3 +1,6 @@
+# Copyright 2019, Oracle Corporation and/or its affiliates. All rights reserved.
```
Comment (ddsharpe) [inline core/src/test/resources/encryption-test.yaml]: These domain YAML files do NOT support comments or blank lines.

File: core/src/test/resources/quote-test.yaml [hunk 3]
```diff
@@ -1,3 +1,6 @@
+# Copyright 2019, Oracle Corporation and/or its affiliates. All rights reserved.
```
Comment (ddsharpe) [inline core/src/test/resources/quote-test.yaml]: These domain YAML files do NOT support comments or blank lines.

File: core/src/test/resources/simple-demo-jms-full.yaml [hunk 4]
```diff
@@ -1,3 +1,6 @@
+# Copyright 2019, Oracle Corporation and/or its affiliates. All rights reserved.
```
Comment (ddsharpe) [inline core/src/test/resources/simple-demo-jms-full.yaml]: These domain YAML files do NOT support comments or blank lines.

File: core/src/test/resources/simple-demo-jms.yaml [hunk 5]
```diff
@@ -1,3 +1,6 @@
+# Copyright 2019, Oracle Corporation and/or its affiliates. All rights reserved.
```
Comment (ddsharpe) [inline core/src/test/resources/simple-demo-jms.yaml]: These domain YAML files do NOT support comments or blank lines.

File: core/src/test/resources/simple-model.yaml [hunk 6]
```diff
@@ -1,3 +1,6 @@
+# Copyright 2019, Oracle Corporation and/or its affiliates. All rights reserved.
```
Comment (ddsharpe) [inline core/src/test/resources/simple-model.yaml]: These domain YAML files do NOT support comments or blank lines.

File: core/src/test/resources/unit-test.yaml [hunk 7]
```diff
@@ -1,3 +1,6 @@
+# Copyright 2019, Oracle Corporation and/or its affiliates. All rights reserved.
```
Comment (ddsharpe) [inline core/src/test/resources/unit-test.yaml]: These domain YAML files do NOT support comments or blank lines.

File: core/src/test/resources/variable_insertion.yaml [hunk 8]
```diff
@@ -1,3 +1,6 @@
+# Copyright 2019, Oracle Corporation and/or its affiliates. All rights reserved.
```
Comment (ddsharpe) [inline core/src/test/resources/variable_insertion.yaml]: These domain YAML files do NOT support comments or blank lines.

File: core/src/test/resources/variables-test.yaml [hunk 9]
```diff
@@ -1,3 +1,6 @@
+# Copyright 2019, Oracle Corporation and/or its affiliates. All rights reserved.
```
Comment (ddsharpe) [inline core/src/test/resources/variables-test.yaml]: These domain YAML files do NOT support comments or blank lines.

File: core/src/test/resources/variablestest.yaml [hunk 10]
```diff
@@ -1,3 +1,6 @@
+# Copyright 2019, Oracle Corporation and/or its affiliates. All rights reserved.
```
Comment (ddsharpe) [inline core/src/test/resources/variablestest.yaml]: These domain YAML files do NOT support comments or blank lines.

File: samples/docker-domain/simple-topology.yaml [hunk 11]
```diff
@@ -1,3 +1,6 @@
+# Copyright 2017, 2019, Oracle Corporation and/or its affiliates. All rights reserved.
```
Comment (ddsharpe) [inline samples/docker-domain/simple-topology.yaml]: These domain YAML files do NOT support comments or blank lines.

File: core/src/test/resources/encryption-test-variables.yaml (hunk 1 above)
Comment (rosemarymarano) [reply core/src/test/resources/encryption-test-variables.yaml]: Removed comments and blank lines.

File: core/src/test/resources/encryption-test.yaml (hunk 2 above)
Comment (rosemarymarano) [reply core/src/test/resources/encryption-test.yaml]: Removed comments and blank lines.

File: core/src/test/resources/quote-test.yaml (hunk 3 above)
Comment (rosemarymarano) [reply core/src/test/resources/quote-test.yaml]: Removed comments and blank lines.

File: core/src/test/resources/simple-demo-jms-full.yaml (hunk 4 above)
Comment (rosemarymarano) [reply core/src/test/resources/simple-demo-jms-full.yaml]: Removed comments and blank lines.

File: core/src/test/resources/simple-demo-jms.yaml (hunk 5 above)
Comment (rosemarymarano) [reply core/src/test/resources/simple-demo-jms.yaml]: Removed comments and blank lines.

File: core/src/test/resources/unit-test.yaml (hunk 7 above)
Comment (rosemarymarano) [reply core/src/test/resources/unit-test.yaml]: Removed comments and blank lines.

File: core/src/test/resources/variable_insertion.yaml (hunk 8 above)
Comment (rosemarymarano) [reply core/src/test/resources/variable_insertion.yaml]: Removed comments and blank lines.

File: core/src/test/resources/variables-test.yaml (hunk 9 above)
Comment (rosemarymarano) [reply core/src/test/resources/variables-test.yaml]: Removed comments and blank lines.

File: core/src/test/resources/variablestest.yaml (hunk 10 above)
Comment (rosemarymarano) [reply core/src/test/resources/variablestest.yaml]: Removed comments and blank lines.

File: samples/docker-domain/simple-topology.yaml (hunk 11 above)
Comment (rosemarymarano) [reply samples/docker-domain/simple-topology.yaml]: Removed comments and blank lines.

**Pull request merged**